.coafile

```diff
@@ -13,6 +13,7 @@ link_ignore_list =
 # clbin.com doesnt work in link_ignore_list
 # See https://github.com/coala/coala-bears/issues/2126
 link_ignore_regex = clbin.com
+use_spaces = True
```

Comment:
This change does not belong in the " .travis.yml: Remove GitHub merge commit workaround" commit - it is part of the workaround, I guess.

Reply:
I've raised bug https://github.com/coala/coala-bears/issues/2949 about this